src/components/visualizations/common/VisualizationTypes.js

```diff
@@ -0,0 +1,9 @@
+// (C) 2007-2018 GoodData Corporation
+export const TABLE = 'table';
+export const COLUMN_CHART = 'column';
+export const BAR_CHART = 'bar';
+export const LINE_CHART = 'line';
+export const PIE_CHART = 'pie';
+export const DOUGHNUT_CHART = 'doughnut';
+export const HEADLINE = 'headline';
+export const CHART_TYPES = [TABLE, COLUMN_CHART, BAR_CHART, LINE_CHART, PIE_CHART, HEADLINE];
```

Comment:
DOUGHNUT_CHART is missing ? why ?

Reply:
@ivan-nejezchleb could you please review it?

Reply:
Whole this PR should be updated according current indigo state - added Area chart, which is missing here now
Doughnut chart is here, because someone wanted to be prepared for future :)